src/main/java/dansplugins/factionsystem/objects/domain/Gate.java

```diff
@@ -95,9 +95,9 @@ public Map<String, String> save() {
         saveMap.put("open", String.valueOf(open));
         saveMap.put("vertical", String.valueOf(vertical));
         saveMap.put("material", material.name());
-        saveMap.put("world", coord1.getWorld());
-        saveMap.put("coord1", coord1.toString());
-        saveMap.put("coord2", coord2.toString());
+        if (coord1 != null) saveMap.put("world", coord1.getWorld());
```

Comment:
When loading gates, we should delete the gate if one of the coordinates is null.

Reply:
Let's also log this incident in the plugin's logger so that server owners are aware of why their gates disappeared.

Reply:
Hey @ConfusedAlex! If you don't have time to address these comments would you mind giving me access to your fork? I'd love to get these fixes merged :)

Reply:
I've invited you :)

Reply:
This can be done another time. This resolves the problem for now. Thank you for your contribution.

Reply:
I opened an issue for this here: #1530